airqo-platform/AirQo-api
# Ft dev k8s manifests

**_WHAT DOES THIS PR DO?_**
- [keep boot disk on instance deletion](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/a4251707acbb11ca534eb2281f221c5a8f837ad3)
- [allow stopping to update](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/a67cd572f2c7b6c301368f0224d36d6aa18a812a)
- [update config servers to 'e2-standard-2'](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/4ba14dcab7cf21af03ad7b91995cb13bf0659bcb)
- [Remove dev sensitive stateful resources](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/35c1f7884d6c0667519815308a665d08da6e1ffb) from terraform
- Add [dev nginx virtual servers and global config](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/fc087057933b3332a12df99c671b2c8ef9d7353f)
- [add nginxinc helm chart installation files](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/80d99974962d59dfc9457a8a4bc106d28d566b50)
- [custom values for nginx installation](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/214eab6e21149239316acf09d7b8678eabfcfba2)
- [switch docs subdomains to wiki](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/bd663a8b432a452925f4de7151f745b2146c223a)
- Add [tls-secret template](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/5c842e8f50fbf1998fb3dcc0387ad4cab08d90af) for nginx

**_HOW DO I TEST OUT THIS PR?_**
- do a static code analysis
- ensure that all the checks pass [especially the one of Terraform](https://app.terraform.io/app/AirQo/workspaces/airqo-250220/runs/run-3EbRRxLgqrovcfVh)
- checkout the docs subdomains and ensure you can securely access them:
  - https://wiki.airqo.net/
  - https://staging-wiki.airqo.net/
  - https://dev-wiki.airqo.net/ (available depending on what time you check)

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_ARE THERE ANY RELATED PRs?_**
- https://github.com/airqo-platform/AirQo-frontend/pull/1316
